bindings/python/py_src/tokenizers/implementations/bert_wordpiece.py

```diff
@@ -115,4 +115,4 @@ def train(
         )
         if isinstance(files, str):
             files = [files]
-        self._tokenizer.train(trainer, files)
+        self._tokenizer.train(files, trainer=trainer)
```

Comment:
As long as we're breaking signature, I would argue we have a different signature like

`train(files, options=bpe_train_options)`, or `train(files, vocab_size=X, ....)` what do you think ?

I like the second version better, the only trouble is the exact description of those options if going to get fuzzy pretty fast and error handling a bit hard. But it "feels" more pythonic, what do you think ?

Either that, or if we keep the `trainer` concept, we should stick to something closer to Rust, with `trainer.train(tokenizer, files)` I actually like that last version better at this moment, the control flow feels more natural.